backend/graph/api/episodes.resolvers.go

```diff
@@ -18,10 +18,30 @@ import (
 	"github.com/bcc-code/bcc-media-platform/backend/graph/api/model"
 	"github.com/bcc-code/bcc-media-platform/backend/user"
 	"github.com/bcc-code/bcc-media-platform/backend/utils"
+	"github.com/google/uuid"
 	"github.com/samber/lo"
 	null "gopkg.in/guregu/null.v4"
 )
 
+// Episode is the resolver for the episode field.
+func (r *chapterResolver) Episode(ctx context.Context, obj *model.Chapter) (*model.Episode, error) {
+	if obj.Episode != nil && obj.Episode.ID != "" {
+		return r.QueryRoot().Episode(ctx, obj.Episode.ID, nil)
+	}
+	tmID := utils.AsUuid(obj.ID)
+	if tmID == uuid.Nil {
+		return nil, nil
+	}
+	epID, err := r.Loaders.TimedMetadataEpisodeIDLoader.Get(ctx, tmID)
+	if err != nil {
+		return nil, err
+	}
+	if epID != nil {
+		return r.QueryRoot().Episode(ctx, fmt.Sprint(*epID), nil)
+	}
+	return nil, nil
+}
+
```

Comment:
```suggestion
// Episode is the resolver for the episode field.
func (r *chapterResolver) Episode(ctx context.Context, obj *model.Chapter) (*model.Episode, error) {
	if obj.Episode != nil && obj.Episode.ID != "" {
		ep, err := r.Loaders.EpisodeLoader.Get(ctx, utils.AsInt(obj.Episode.ID))
		return model.EpisodeFrom(ctx, ep), err
	}
	tmID := utils.AsUuid(obj.ID)
	if tmID == uuid.Nil {
		return nil, nil
	}
	epID, err := r.Loaders.TimedMetadataEpisodeIDLoader.Get(ctx, tmID)
	if err != nil {
		return nil, err
	}
	if epID != nil {
		ep, err := r.Loaders.EpisodeLoader.Get(ctx, *epID)
		return model.EpisodeFrom(ctx, ep), err
	}
	return nil, nil
}
```

I think we should use loaders here otherwise we bypass all aggregation and caching.